wire-ios-data-model/sourcery/AutoMockable.stencil

```diff
@@ -0,0 +1,209 @@
+// swiftlint:disable line_length
```

Comment:
This stencil is essentially https://github.com/krzysztofzablocki/Sourcery/blob/master/Templates/Templates/AutoMockable.stencil but has some minor formatting and naming modifications.